.github/workflows/gradle.yml

```diff
@@ -35,6 +35,7 @@ jobs:
           ./gradlew runFunctionalTest
           javac -cp api/build/libs/minio-${DEV_VERSION}-all.jar functional/TestUserAgent.java
           java -Dversion=${DEV_VERSION} -cp api/build/libs/minio-${DEV_VERSION}-all.jar:functional TestUserAgent
+          ./gradlew clean
```

Comment:
What problem did you see to add this line?

Reply:
This is mainly a problem that happens sporadically causing TestUserAgent.java build to fail.